src/generated/type/src/client.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 Google LLC
```

Comment:
This should likely not exist?

Reply:
Yes, but all `.rs.mustache` files are generated unconditionally.  I could not think of a simple trick to avoid the code generation.  Maybe the Codec should return a list of input templates -> output file names?

Reply:
Fair, maybe just open an issue for now.

Reply:
Tracking in #357